packages/SwingSet/misc-tools/replay-transcript.js

```diff
@@ -77,11 +78,9 @@ async function makeBundles() {
     import.meta.url,
   );
 
+  const lockdown = await bundleSource(entryPaths.lockdown, 'nestedEvaluate');
```

Comment:
Any particular reason to peek into `entryPaths` here? It seems like `getLockdownBundle()` should work?

Reply:
I'll add a comment.. if you're running that function, you're probably hacking on the contents of lockdown or supervisor/liveslots, and you're going to want to re-build the bundles each time you run the program. Showing the long form serves as a reminder of what you'd need to do.